src/main/kotlin/no/skatteetaten/aurora/boober/feature/DeploymentConfigFeature.kt

```diff
@@ -196,4 +205,11 @@ class DeploymentConfigFeature : Feature {
 
         return mapOf("deployTag" to adc.dockerTag).addIfNotNull(pauseLabel).normalizeLabels()
     }
+
+    private fun createKlientIDHeader(adc: AuroraDeploymentSpec): String {
+        // TODO: Muligens en for dyr operasjon
```

Comment:
Fant du ut om det var en dyr operasjon?

Reply:
Det er flere features som kaller cantus. Burde vi vurdere å la boober gjøre ett kall til getImageInformation(), og gjenbruke samme resultatet?

Reply:
Det er vel ikke noen som gjør akkurat dette kallet? Det gjøres for job, fluentbit og toxiproxy. Men ingen av de resolver applikasjonstaggen

Reply:
Med mindre jeg overser noe, men er det ikke samme manifestet som blir hentet i alle featurene?

Reply:
toxiproxy og fluentbit kan vi jo holde utenfor.

Reply:
Ok, var litt kjapp her. Det er kanskje ingen andre features som henter manifestet til selve applikasjonen.

Reply:
Går bort fra å bruke cantus til å bruke APP_VERSION miljøvariabelen som settes i alle images av Architect.